security/pkg/pki/ca/ca.go

```diff
@@ -32,12 +32,20 @@ const (
 	// istioCASecretType is the Istio secret annotation type.
 	istioCASecretType = "istio.io/ca-root"
 
-	// cACertChainID is the CA certificate chain file.
-	cACertID = "ca-cert.pem"
-	// cAPrivateKeyID is the private key file of CA.
-	cAPrivateKeyID = "ca-key.pem"
-	// cASecret stores the key/cert of self-signed CA for persistency purpose.
-	cASecret = "istio-ca-secret"
+	// caCertID is the CA certificate chain file.
+	caCertID = "ca-cert.pem"
```

Comment:
Why do you change "cA" to "ca"? We have a couple of places that are using "ca", but I think the correct way is to change them to "cA", not the other way around. If you are following some golang format standard, please let me know.

Reply:
Go recommends using all capital or lower case for acronyms.
https://github.com/golang/go/wiki/CodeReviewComments#initialisms